microsoft/utils/CreateServicePrincipal_linux.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# This script will create Azure Active Directory application, and Service Principal that can access Azure resources
+# It will return clientID, tenantID, client secret
+# NOTE: this version is for Linux users
+
+printf "Starting steps of creating service principal...\n"
+read -rp "Enter subscription Id: " subscriptionId
+
+az login && az account set --subscription "$subscriptionId" && echo "Successfully login with subscription $subscriptionId"
+
+defaultIdentifier="LISAv2"$(uuidgen)
```

Comment:
Remove "V2" from the name, if it doesn't break anything.

Reply:
it's just a name, chould be anything with valid format
removed